MadWorkshopFunction/src/main/java/app/App.java

```diff
@@ -19,6 +19,11 @@
 public class App implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
 
     public APIGatewayProxyResponseEvent handleRequest(final APIGatewayProxyRequestEvent input, final Context context) {
+        try {
+            final com.amazonaws.services.s3.AmazonS3 s3Client = 
+                com.amazonaws.services.s3.AmazonS3ClientBuilder.standard().withRegion("ap-northeast-1").build();
+            final com.amazonaws.services.s3.model.ObjectListing files = s3Client.listObjects("dummy-bucket");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code is manually checking for the existence of a bucket in S3. You could use the SDK (operation name: [doesBucketExist](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/AmazonS3Client.html#doesBucketExist-java.lang.String-) or [doesBucketExistV2](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/AmazonS3Client.html#doesBucketExistV2-java.lang.String-)) to automatically check for this instead.